main/data/theme.css

```diff
@@ -110,10 +110,6 @@ window.dino-main button.dino-chatinput-button:checked:backdrop {
 }
 
 
-.dino-chatinput textview, .dino-chatinput textview text {
-    background-color: transparent;
```

Comment:
Could you try to set the `background-color` to `@theme_base_color` instead of removing it?

Reply:
It works :raised_hands: